src/hmpb/findTargets.ts

```diff
@@ -12,7 +15,12 @@ export interface TargetShape {
 export default function* findTargets(
   ballotImage: ImageData,
   bounds: Rect,
-  { aspectRatio = 1.5, aspectRatioTolerance = 0.1 } = {}
+  {
+    aspectRatio = 1.5,
+    aspectRatioTolerance = 0.1,
+    widthPercentage = 0.085,
+    widthPercentageTolerance = 0.005,
```

Comment:
is this tolerance too tight? Does it need to be this tight?

Reply:
It's pretty tight, but this only really happens on the templates anyway so it's pretty clean.